apps/examples/testcase/le_tc/network/tc_net_getsockopt.c

```diff
@@ -165,6 +165,7 @@ static void tc_net_getsockopt_optval_n(int s)
    * @precondition	   :socket file descriptor
    * @postcondition	   :
    */
+#if 0 // get has lwip_tcp in the accepticonn option and not in set.
```

Comment:
@pillip8282 Could you review this?

Reply:
It's already commented  from being called. check below. This change is just  to commen the unused definition.
https://github.com/Samsung/TizenRT/blob/094c3a74808b66d320c7513e00823e38c9ebab6a/apps/examples/testcase/le_tc/network/tc_net_getsockopt.c#L253